pristine-zen-garden.html

```diff
@@ -87,21 +84,16 @@ <h3>Requirements</h3>
                     By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.
                 </p>
             </div>
-
             <footer>
                 <a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
                 <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
                 <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Creative Commons license of this site: Attribution-NonCommercial-ShareAlike." class="zen-license">CC</a>
                 <a href="http://mezzoblue.com/zengarden/faq/#aaa" title="Read about the accessibility of this site" class="zen-accessibility">A11y</a>
                 <a href="https://github.com/mezzoblue/csszengarden.com" title="Fork this site on Github" class="zen-github">GH</a>
             </footer>
-
         </div>
-
-
         <aside class="sidebar" role="complementary">
             <div class="wrapper">
-
                 <div class="design-selection" id="design-selection">
                     <h3 class="select">Select a Design:</h3>
                     <nav role="navigation">
```

Comment:
Some of the list items as in the following lines have extra spaces between the tags. I don't think has any affect but could be easier to read without them